apps/stable_diffusion/src/utils/stable_args.py

```diff
@@ -341,5 +349,37 @@ def path_expand(s):
     action=argparse.BooleanOptionalAction,
     help="Apply Winograd on selected conv ops.",
 )
+##############################################################################
```

Comment:
These are still used for SD? Can we remove them? They aren't used in generate_sharktank.py anymore